benchexec/tools/veriabs.py

```diff
@@ -0,0 +1,73 @@
+"""
+BenchExec is a framework for reliable benchmarking.
+This file is part of BenchExec.
+
+Copyright (C) 2007-2015  Dirk Beyer
+All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+import logging
+import xml.etree.ElementTree as ET
+
+import benchexec.util as util
+import benchexec.tools.template
+import benchexec.result as result
+
+class Tool(benchexec.tools.template.BaseTool):
+    """
+	veriabs for SV-COMP 2017
+    """
+    def executable(self):
+        return util.find_executable('veriabs')
+
+
+    def version(self, executable):
+        return " Version 0.001 Alpha"
+
+
+    def name(self):
+        return 'VeriAbs SV-COMP 2017'
+
+
+    def cmdline(self, executable, options, tasks, propertyfile, rlimits):
+        if propertyfile:
+            options += ['--property-file', propertyfile]
+        self.options = options
+        return [executable] + options + tasks
+
+
+
+
+    def determine_result(self, returncode, returnsignal, output, isTimeout):
+        lines = " ".join(output[-10:])
+        print(lines)
+
+        if isTimeout:
+            return 'TIMEOUT' 
+
+        if "INVALID-POINTER" in lines or "DYNAMIC_OBJECT" in lines or "dereference failure" in lines:
+            return result.RESULT_FALSE_DEREF		
+        elif "SUCCESS" in lines:
+            return result.RESULT_TRUE_PROP
+        elif "FAILED" in lines:
+            return result.RESULT_FALSE_REACH
```

Comment:
Can the tool answer only with "false deref", "true", or "false reach"? No other properties supported, like e.g. "false memleak" etc.?